src/main/java/org/opensearch/neuralsearch/processor/chunker/DelimiterChunker.java

```diff
@@ -25,9 +27,18 @@ public final class DelimiterChunker implements Chunker {
     /** The default delimiter value used when none is specified. Uses two consecutive newline characters to split on paragraph boundaries. */
     public static final String DEFAULT_DELIMITER = "\n\n";
 
+    /** The parameter field name for specifying the chunk size. */
+    public static final String CHUNK_SIZE_FIELD = "chunk_size";
+
+    /** The default chunk size value. */
+    public static final int DEFAULT_CHUNK_SIZE = 0;
```

Comment:
Could be better to set it as Integer.MAX_VALUE?

Reply:
I agree.